### DISCUSSION TITLE
Driver unknown

### DISCUSSION THREAD

**Discussion by RGMacFarland** [feelfreelinux/octo4a#62] [Q&A] [answered]
I've installed 4a and it seems to be almost working. I can see on my phone that my Prusa Mini is connected to /dev/bus/usb/001/002 but in Octoprint it won't connect. On the phone it give me an error on what driver to use. This is under the Printer is connected..  I have 5 options. FTDI, CDC/ACM, CH341, Prollific and CP21xx. Which one should I pick?

**Comment by feelfreelinux** [comment, marked as answer]
I'm pretty sure that for prusa mini it's supposed to be CDC/ACM

**Reply by RGMacFarland** [reply]
Awesome,

That was it. As soon as i changed to CDC/ACM it connected.

Thanks for the quick response.

**Reply by brolly916** [reply]
> I'm pretty sure that for prusa mini it's supposed to be CDC/ACM

hello @feelfreelinux  can you kindly advise which driver I need to choose for my CR10 S5 please ? as i am able to choose the /dev/ttyOcto4a. however octoprint is still not able to connect / access my printer

**Reply by solo2424** [reply]
@feelfreelinux I am having the same issue with MakerBot Replicator 2X. Do you have any suggestions? I was able to hook my Ender 3 Pro up with no issues.

**Reply by Saurabh251-cmd** [reply]
CH341

**Comment by brolly916** [comment]
I have the same issue with my CR10 S5 can someone tell me which driver i need to choose please ?

**Reply by Saurabh251-cmd** [reply]
CH341
